test/registration/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ PCL_ADD_TEST(registration_warp_api test_warps
 
 PCL_ADD_TEST(correspondence_estimation test_correspondence_estimation
              FILES test_correspondence_estimation.cpp
-             LINK_WITH pcl_gtest pcl_registration pcl_features)
+             LINK_WITH pcl_gtest pcl_io pcl_registration pcl_features)
```

Comment:
I assume this change (and the one above with `PCL_SET_TEST_DEPENDENCIES`) are necessary because test_correspondence_estimation.cpp includes pcd_io.h? It seems like including pcd_io.h is not actually necessary there, so we can instead remove that include and leave this CMakeLists.txt unchanged.